_pages/acknowledgements.md

```diff
@@ -3,6 +3,16 @@
 
 # Acknowledgements
 
-This work has been supported by the [International Neuroinformatics Coordinating Facility](https://www.incf.org/){:target="_blank"}, the [Neuroimaging Data Sharing Task Force](https://web.archive.org/web/20170813183704/http://wiki.incf.org/mediawiki/index.php/Neuroimaging_Task_Force){:target="_blank"}, the Laura and John Arnold Foundation, NIMH Grant R24MH114705, and NSF grant OAC-1760950.
+This work has been done by the community of [individual contributors](https://bids-specification.readthedocs.io/en/stable/appendices/contributors.html). Their work was
+supported by the
+
+Funding Source | Target
+----- | -----
+[International Neuroinformatics Coordinating Facility](https://www.incf.org/){:target="_blank"} | [Neuroimaging Data Sharing Task Force](https://web.archive.org/web/20170813183704/http://wiki.incf.org/mediawiki/index.php/Neuroimaging_Task_Force){:target="_blank"} collated initial group of people to push pilot BIDS development and promotion
+Laura and John Arnold Foundation |
+[NIMH R24MH114705](https://reporter.nih.gov/project-details/9411944) | BIDS common derivatives BEP
+[NSF OAC-1760950](https://www.nsf.gov/awardsearch/showAward?AWD_ID=1760950) | 
+[NIMH 1R24MH117295](https://reporter.nih.gov/project-details/9795271) | DANDI archive contributed development of Microscopy BEP
+[NIMH 2P41EB019936](https://reporter.nih.gov/project-details/10334133) | ReproNim project contributed various enhancements and fixes
 
```

Comment:
```suggestion
[NIMH MH117179](https://reporter.nih.gov/project-details/10145071) | OpenNeuro archive contributed various enhancements and fixes
[NIMH 1ZIAMH002977-01](https://reporter.nih.gov/project-details/10489085) |  OpenNeuroPET project contributed various enhancements and fixes 
[NIMH 5R01MH126699-02](https://reporter.nih.gov/project-details/10460628) | Brain Connectivity BEP
[NIMH MH109682](https://reporter.nih.gov/project-details/9982125) | NeuroScout
[Novo Nordisk NNF20OC0063277]() | 
```

Reply:
Could you also please add:

[NIMH 1RF1MH126700-01A1](https://reporter.nih.gov/project-details/10480619] Hierarchical Event Descriptors (HED)

Thx